diff -urN a/HxCFloppyEmulator/libhxcfe/trunk/sources/floppy_loader.c b/HxCFloppyEmulator/libhxcfe/trunk/sources/floppy_loader.c
--- a/HxCFloppyEmulator/libhxcfe/trunk/sources/floppy_loader.c	2022-06-07 20:02:57.055255975 +1000
+++ b/HxCFloppyEmulator/libhxcfe/trunk/sources/floppy_loader.c	2022-06-07 20:06:10.120092735 +1000
@@ -128,7 +128,7 @@
 #elif __APPLE__
 		hxcfe_execScriptLine( hxcfe, "set SPSCAPS_LIB_NAME CAPSimage.framework/CAPSimage" );
 #else
-		hxcfe_execScriptLine( hxcfe, "set SPSCAPS_LIB_NAME libcapsimage.so.5.1" );
+		hxcfe_execScriptLine( hxcfe, "set SPSCAPS_LIB_NAME /app/extensions/capslib/extra/lib/libcapsimage.so.5.1" );
 #endif
 
 		hxcfe_execScriptFile(hxcfe, "config.script");
